route/grafananet.go

```diff
@@ -249,6 +251,8 @@ func (route *GrafanaNet) flush(shard int) {
 				buf := make([]byte, 300)
 				n, _ := resp.Body.Read(buf)
```

Comment:
if we're going to read all of the body anyway, might as well replace this line with the ReadAll (and then display the first 300 chars, or len of the full body, whichever is lower)

Reply:
Why read all of it into memory if we only use the first 300 bytes?

Reply:
from what I can tell, ReadAll already reads all of it in memory (and then discards the buffer), so the suggestion is to first use it to log the first 300 chars, then discard it.
but you know, it wouldn't surprise me if the go compiler gets sufficiently smart to detect we discard the buffer, and optimizes the ReadAll to just discard all data immediately instead of first storing it in a slice, so let's keep what you have.